boards/arm/nrf52833_pca10111/nrf52833_pca10111.dts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/dts-v1/;
+#include <nordic/nrf52833_qiaa.dtsi>
+
+/ {
+	model = "Nordic PCA10111";
+	compatible = "nordic,pca10111", "nordic,nrf52833-qiaa",
+		     "nordic,nrf52833";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0_green: led_0 {
+			gpios = <&gpio1 9 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+		led1_red: led_1 {
+			gpios = <&gpio0 11 GPIO_INT_ACTIVE_LOW>;
+			label = "Red LED 1";
+		};
+		led1_green: led_2 {
+			gpios = <&gpio0 15 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 1";
+		};
+		led1_blue: led_3 {
+			gpios = <&gpio0 17 GPIO_INT_ACTIVE_LOW>;
+			label = "Blue LED 1";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio0 10 GPIO_PUD_PULL_UP>;
+			label = "Push button switch 0";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		sw0 = &button0;
+		led0 = &led0_green;
+		led1 = &led1_red;
+		led2 = &led1_green;
+		led3 = &led1_blue;
+		led0-green = &led0_green;
+		led1-red   = &led1_red;
+		led1-green = &led1_green;
+		led1-blue  = &led1_blue;
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <31>;
+	rx-pin = <30>;
+	rts-pin = <29>;
+	cts-pin = <28>;
+};
+
```

Comment:
Nodes `adc`, `i2c0`, `pwm0`, `spi1`, and `spi3` are not enabled here, but the corresponding instances are enabled by default in `Kconfig.defconfig`. I'm not sure what was the actual intent, but since `nrf52833_pca10111.yaml` also mentions that `adc` and `pwm` are supported, I guess those nodes (at least some of them) were to be enabled.

Reply:
Thanks for noting that. I would prefer to disable everything and let application configuration enable things it needs.

Reply:
I will check the Kconfig.defconfig content and resubmit.

Reply:
I have removed all config options that are not needed